### ISSUE TITLE
Impossible to load map after changing settings

### ISSUE THREAD

**Issue by el-srabo** [umap-project/umap#1725]
**Describe the bug**
I changed the "Default view" setting from "Saved center and zoom" to "Latest feature", and ever since the map won't load. The general interface does load but where the map itself should be the area stays gray (see screenshot), and the blue loading bar above keeps on loading.

This is especially embarrassing because when I click on " Edit" the "Settings" icon does not appear, so I cannot change the settings back. Is there any way I could revert the map to an earlier version?

**To Reproduce**
1. Go to https://framacarte.org/en/map/velochness_148987

**Expected behavior**
Map should load.

**Screenshots**
![Screenshot_20240401-231247_Firefox](https://github.com/umap-project/umap/assets/150371715/f9ce5bbc-fbb7-4b6a-91f7-446d7aaf0b37)

**Smartphone (please complete the following information):**
- Device: Fairphone 2
-  OS: LineageOS 18.1 (Android 11)
-  Browser: Firefox
-  Version: 123.0

(I also tried on an iPad with Safari)

**Edit**
I managed to access the settings by editing a data layer, I could therefore fix my map, but the "Latest feature" option definitely seems broken.

**Comment by almet** [comment]
It seems to be working when I'm trying. Did you manage to solve your issue?

**Comment by el-srabo** [comment]
I managed to get back to a functioning map by changing the "Default view" setting back to the option "Saved center and zoom", but the "Latest feature" option never worked for me

**Comment by davidbgk** [comment]
This is something we fixed in https://github.com/umap-project/umap/releases/tag/2.1.3 so I hope it could be closed now. Not sure when Framacarte updated but that instance is now using the latest version.

**Comment by el-srabo** [comment]
The bug where the map did not load is indeed corrected, thanks for that!
However, how is the "latest feature" determined? In my case the map does not zoom into the last marker I put but the penultimate (and even if I put new markers it still zooms into the same one, the one near Midleton).
Maybe there is a delay between the time where a new marker is put and the time where it is detected as "latest feature"? As I write the last marker I put is minutes old, I'll see if the behavior changes later.

**Comment by yohanboniface** [comment]
At this time, the latest marker is determined by the "sort key", or the `name` property if this key is not set. And if there are many layers, it will look into the top layer first, then the layer below, etc. until finding one layer with at least one feature.

**Comment by el-srabo** [comment]
> At this time, the latest marker is determined by the "sort key", or the `name` property if this key is not set. And if there are many layers, it will look into the top layer first, then the layer below, etc. until finding one layer with at least one feature.

I'm not familiar with how umap works under the hood, so I don't really understand why the feature that is opened on start is not the one I last put, even if all features involved (the one that is opened on start and the ones that were added later) belong to the same layer.
I did not set any 'sort key' so based on what you wrote the latest marker should be determined based on the 'name' propery. I assume this looks up the latest marker 'name' that was added?
